### PR TITLE
Updates for Syft 0.14

### PR DESCRIPTION
**Pull request by Serubin**
Update syft digest param for 0.14.0 - now imageID. (This is not to be confused with manifestDigest)

Verified scanning still works on org Serubin https://app.fossa.com/projects/custom%2B13453%2Ferror-test/refs/branch/master/sha256%3A2e6f503b8d20ca49209144d5e564f9d9aea09cce7e752656750ea786e3727004

Our version of syft has also been updated to version v0.14.0 - a new release will need to be cut.

### CODE REVIEW

**Review by scruffystuffs — CHANGES_REQUESTED** [review]
Blocking merge until CI shows the updated release.
The fossas/syft repo will need to cut a new release before this merges.

You can also verify your changes with `fossa container analyze --output IMAGE`

**Comment by Serubin** [comment]
fossas/syft has already cut a new release - https://github.com/fossas/syft/releases/tag/v0.14.0-fossa

Zero diff in `fossa container analyze --output` confirmed

**Comment by scruffystuffs** [comment]
I'm looking at CI logs, and we're still downloading and building 0.9.2.  `vendor_download.sh` is the script that downloads the release, and it points to the latest version.

**Review by scruffystuffs — APPROVED** [review]
Latest build shows the correct version, this is good to go once CI finishes.

**Pull request merged**
